helm-charts/policy/templates/policy_framework_cluster_role.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright Contributors to the Open Cluster Management project
+
+# Licensed Materials - Property of IBM
+# IBM Confidential
+# OCO Source Materials
+# (C) Copyright IBM Corporation 2016, 2019 All Rights Reserved
+# The source code for this program is not published or otherwise divested of its trade secrets, irrespective of what has been deposited with the U.S. Copyright Office.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.org }}:{{ template "policy.fullname" . }}-framework
+  labels:
+    app: {{ template "policy.name" . }}-framework
+    chart: {{ template "policy.chart" . }}
+    component: "policy"
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - namespaces
```

Comment:
What needs to create and delete namespaces?

Reply:
spec-sync and status-sync both have this rule, it looks like status-sync uses it here: https://github.com/stolostron/governance-policy-status-sync/blob/main/main.go#L196

Reply:
I just assumed they were created on cluster import, but that's handy to know. Thanks!